db/migrate/20241024102540_create_lcms_engine_integrations_webhook_configurations.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CreateLcmsEngineIntegrationsWebhookConfigurations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :lcms_engine_integrations_webhook_configurations do |t|
+      t.string :event_name, null: false, index: { name: 'index_webhook_configurations_on_event_name' }
+      t.boolean :active, default: true
```

Comment:
I would add `null: false` as PostgreSQL can have `true|false|null` values in boolean fields. But that's not a critical for now I suppose.

Reply:
thanks, added